shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1696,13 +1697,178 @@ public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOE
         }
     }
 
+    @Data
+    public static final class GetTableKeys implements Request, WireCommand {
```

Comment:
This should probably be called "readTableKeys"

Reply:
done.